src/lightning/pytorch/utilities/model_summary/model_summary.py

```diff
@@ -292,8 +301,24 @@ def _get_summary_data(self) -> List[Tuple[str, List[str]]]:
             arrays.append(("In sizes", [str(x) for x in self.in_sizes]))
             arrays.append(("Out sizes", [str(x) for x in self.out_sizes]))
 
+        total_leftover_params = self.total_parameters - self.total_layer_params
+        if total_leftover_params > 0:
+            self._add_leftover_params_to_summary(arrays, total_leftover_params)
+
         return arrays
 
+    def _add_leftover_params_to_summary(self, arrays: List[Tuple[str, List[str]]], total_leftover_params: int):
+        """Add summary of params not associated with module or layer to model summary."""
+        layer_summaries = dict(arrays)
+        layer_summaries[" "].append(str(len(self._layer_summary) + 1))
```

Comment:
Pretty sure this shouldn't be +1 here. I'm getting non-consecutive numbering here:


```
  | Name         | Type   | Params
----------------------------------------
0 | layer        | Linear | 1.1 K 
1 | layer1       | Linear | 1.1 K 
2 | layer2       | Linear | 66    
4 | other params | n/a    | 3     
----------------------------------------
```

On another note, I would not even put a number in the first place. The layers appear there in the order they were recorded during use in forward. But the "other params" is really not something we can record in order so they are just there at the end as a separate row unrelated to the ordering of others. What do you think?

Reply:
Sounds good. I agree that `other params` should not have an index in the summary.